### PR TITLE
[PyTorch/XLA] Set a lower accuracy for DDP ResNet50

### PR DESCRIPTION
**Pull request by alanwaketan**
# Description

It has been a known issue with the DDP integration that it converges slower than the native optimizer.step() solution.

# Tests

Ran a one shot test to verify that the latest config does apply to the test.

**Instruction and/or command lines to reproduce your tests:**
./scripts/run-oneshot.sh -t pt-2.0-resnet50-pjrt-ddp-conv-v4-8-1vm

**List links for your tests (use go/shortn-gen for any internal link):**
N/A.

# Checklist

Before submitting this PR, please make sure (put X in square brackets):
- [x] I have performed a self-review of my code.
- [x] I have necessary comments in my code, particularly in hard-to-understand areas.
- [x] I have run one-shot tests and provided workload links above if applicable. 
- [x] I have made or will make corresponding changes to the doc if needed.

### CODE REVIEW

**Review by JackCaoG — APPROVED** [review]

File: tests/pytorch/nightly/resnet50-mp.libsonnet [hunk 1]
```diff
@@ -80,6 +80,34 @@ local tpus = import 'templates/tpus.libsonnet';
       },
     },
   },
+  // DDP converges worse than MP.
+  local convergence_ddp = common.Convergence {
```
Comment (will-cromar) [inline tests/pytorch/nightly/resnet50-mp.libsonnet:84]: Can you refactor this to inherit from `convergence`? It would look something like

```
local convergence_ddp = convergence {
  metricConfig+: {
    ...
  }
}
```

File: tests/pytorch/nightly/resnet50-mp.libsonnet [hunk 2]
```diff
@@ -80,6 +80,34 @@ local tpus = import 'templates/tpus.libsonnet';
...
+    ],
+    metricConfig+: {
+      sourceMap+:: {
+        tensorboard+: {
+          aggregateAssertionsMap+:: {
+            'Accuracy/test': {
+              FINAL: {
+                fixed_value: {
+                  comparison: 'GREATER',
+                  value: 65,
```
[the remark this answers is not in the document]
Comment (alanwaketan) [reply tests/pytorch/nightly/resnet50-mp.libsonnet:99]: Agreed. On the other hand, it also feels like we need customer request to justify the investment on the investigation.
